data/amazon-simpledb/src/main/java/com/teiid/spring/data/amazon/simpledb/SimpleDBConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teiid.spring.data.amazon.simpledb;
+
+import org.teiid.translator.simpledb.api.BaseSimpleDBConfiguration;
+
+public class SimpleDBConfiguration implements BaseSimpleDBConfiguration {
+    private String accessKey;
+    private String secretKey;
+
+    @Override
+    public String getAccessKey() {
+        return accessKey;
+    }
+
+    public void setAccessKey(String accessKey) {
+        this.accessKey = accessKey;
+    }
+
+    @Override
+    public String getSecretAccessKey() {
+        return secretKey;
+    }
+
```

Comment:
Needs a getSecretKey() method here.

Reply:
Since, there was a default method, we don't need to implement it in SimpleDBConfiguration class.
It will automatically call getSecretAccessKey whenever we call getSecretKey method.
So, it's not required.

Reply:
I think last time I did that the spring introspection did not behave well, as it looks for bot get and set methods and did not find one from default I belive. Also see the latest change that @shawkins did on Teiid side on this too

Reply:
@rareddy I updated the code, and the build is now passing. Can we move towards closing the PR?

cc: @shawkins